WMF Framework/UITraitCollection+Extensions.swift

```diff
@@ -0,0 +1,7 @@
+import UIKit
+
+public extension UITraitCollection {
+    var wmf_preferredContentSizeCategory: UIContentSizeCategory {
```

Comment:
I kept it because I also thought I was missing something, but I just deleted it and tested the CollectionViewCells that used it, and they are responding to trait changes. So maybe there was something more to it, and it was lost over time.